docs/root/configuration/advanced/well_known_filter_state.rst

```diff
@@ -56,6 +56,11 @@ The following list of filter state objects are consumed by Envoy extensions:
   <envoy_v3_api_field_extensions.filters.network.tcp_proxy.v3.TcpProxy.tunneling_config>` to disable tunneling on a
   per-connection bases. Accepts values "true" and "false".
 
+``envoy.filters.network.http_connection_manager.local_reply_owner``
+  Shared filter status for logging which filter in the HTTP filter chain sent the local reply. Fields:
+
+  * ``filter_config_name``: filter configuration name in the HTTP filter chain;
```

Comment:
I don't think you implemented the fields, and we probably don't need it for now. Maybe just say this usually serializes to the filter config name?